cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,27 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignUpPageObject from '../support/pages/signUp.pageObject';
+
+const signUpPage = new SignUpPageObject();
+
 describe('User', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
-
+  it('should be able to follow the another user', () => {
+    signUpPage.visit();
+    signUpPage.typeUsername(user.username);
+    signUpPage.typeEmail(user.email);
+    signUpPage.typePassword(user.password);
+    signUpPage.clickSignUpBtn();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(1000);
```

Comment:
Using `cy.wait(1000)` is generally discouraged as it can lead to flaky tests. It's better to wait for specific elements or conditions using commands like `cy.get()` or `cy.contains()` with `.should('be.visible')` to ensure the application is in the expected state before proceeding.